build.sbt

```diff
@@ -49,7 +49,7 @@ lazy val scoverageSettings: Seq[Setting[_]] = Seq(
     ".*(BuildInfo|Routes|Options).*",
     "logger.*\\(.*\\)"
   ).mkString(";"),
-  coverageMinimum := 96,
+  coverageMinimum := 95.8,
```

Comment:
Controversial. I'm started to get tired of having to seek out and add additional tests in areas unrelated to changes which I've made in order to keep the coverage above 96%. I don't believe this is time well spent and certainly not at this late stage in the project.

Reply:
I'm happy with this all the way down to the mid 70s.